pkg/webhook/admission/http_test.go

```diff
@@ -151,16 +104,28 @@ type nopCloser struct {
 func (nopCloser) Close() error { return nil }
 
 type fakeHandler struct {
-	invoked bool
-	fn      func(context.Context, atypes.Request) atypes.Response
+	invoked        bool
+	fn             func(context.Context, Request) Response
+	decoder        *Decoder
+	injectedString string
```

Comment:
Is it actually tested anywhere?

Reply:
yep, pkg/webhook/admission/webhook_test.go